blog-mysql-seed/mysql/mysql.star

```diff
@@ -0,0 +1,52 @@
+MYSQL_IMAGE = "mysql:8.0.32"
+
+def create_database(plan, database_name, database_user, database_password, seed_script_artifacts = []):
+    files = {}
+    for index, artifact in enumerate(seed_script_artifacts):
+        files["/docker-entrypoint-initdb.d/{}.sql".format(index)] = artifact
+    service_name = "mysql-{}".format(database_name)
+    mysql_service = plan.add_service(
+        service_name = service_name,
+        config = ServiceConfig(
+            image = MYSQL_IMAGE,
+            ports = {
+                "db": PortSpec(
+                    number = 8080,
+                    transport_protocol = "TCP",
+                    application_protocol = "http",
+                ),
+            },
+            files = files,
+            env_vars = {
+                "MYSQL_ROOT_PASSWORD": "root",
+                "MYSQL_DATABASE": database_name,
+                "MYSQL_USER": database_user,
+                "MYSQL_PASSWORD":  database_password,
+            },
+        )
+    )
+    # Wait for MySQL to become available
+    mysql_flags = ["-u", database_user, "-p{}".format(database_password), database_name]
```

Comment:
Also wondering why we have such a split here between this and the singleton `["mysql"]` below, feels weird at first glance

Reply:
Done